cmdstanpy/model.py

```diff
@@ -257,7 +257,7 @@ def compile(self, opt_lvl: int = 3, force: bool = False) -> None:
                         do_command(cmd, cmdstan_path(), logger=self._logger)
                     except RuntimeError as e:
                         self._logger.error(
-                            'file %s, exception %s', stan_file, repr(e)
+                            'file %s, exception %s', stan_file, e
```

Comment:
https://stackoverflow.com/a/1438297 - suggests that fix is changing `repr` to `str` - does this do the right thing w/r/t newlines?

Reply:
I think leaving it as `e` is the same as `str(e)`, but I can change it to `str(e)` if you want. 

> does this do the right thing w/r/t newlines?

Yes, before, `repr` just escaped all the special characters, so newline became "\n" for example. The `str` does escape newlines, and that's what we want here.

Reply:
great, please use str(e)

Reply:
done